--- Guile-0.001/Makefile.PL.orig	Sat Jul 21 20:28:01 2001
+++ Guile-0.001/Makefile.PL	Wed Dec 25 00:02:24 2002
@@ -57,6 +57,9 @@
     next unless -e "$dir/libguile.h" and -d "$dir/libguile";
     $include_dir = $dir;
   }
+  if (not defined $include_dir and -e "/usr/include/libguile.h" and -d "/usr/include/libguile") {
+    $include_dir = '/usr/include';
+  }
   if (not defined $include_dir) {
     die "Unable to find libguile.h and libguile/*.h.  I checked all the directories mentioned by your \"guile-config\" to no avail.  Giving up!\n";
   }
